### PR TITLE
Applied MVP to the base project

### PR DESCRIPTION
**Pull request by AndresConfortiGL**
## Description
Second PR - Added the MVP structure to the base project.

## Story Type
- New feature :white_check_mark:
- Change request
- Bug fix
- Tech

## How Can This Be Tested?
By opening the app, you will see the changes made to the project structure are working.

## Evidences
| Nexus S  | Pixel 3a |
| ------------- | ------------- |
| Nexus S - API 26 | Pixel 3a - Api 26 |

| Video evidence on Nexus S | Video evidence on Pixel 3a |
| ------------- | ------------- |
|ezgif com-gif-maker||

## Tested on
The project was tested on Android 8.0 / API 26 using the following devices:

- Nexus S
- Pixel 3a

## Package Structure
| Screenshot  |
| ------------- |
| structure1 |

### CODE REVIEW

File: app/src/main/java/activity/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.mobile.tandil.javabaseproject.activity;
+
+import android.os.Bundle;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.mobile.tandil.javabaseproject.databinding.ActivityMainBinding;
+
```
Comment (jnicolaslaurenti) [inline app/src/main/java/activity/MainActivity.java]: remove empty lines please

File: app/src/main/java/mvp/view/ParkingView.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package mvp.view;
+
+import android.app.Activity;
+
```
Comment (jnicolaslaurenti) [inline app/src/main/java/mvp/view/ParkingView.java]: same here

File: app/src/main/java/activity/MainActivity.java (hunk 1 above)
Comment (AndresConfortiGL) [reply app/src/main/java/activity/MainActivity.java]: done

File: app/src/main/java/mvp/view/ParkingView.java (hunk 2 above)
Comment (AndresConfortiGL) [reply app/src/main/java/mvp/view/ParkingView.java]: done

**Review by jnicolaslaurenti — APPROVED** [review]

**Review by Georgina-Fernandez — APPROVED** [review]

File: app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/ParkingView.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package com.mobile.tandil.javabaseproject.mvp.view;
+
+import android.app.Activity;
+
+import com.mobile.tandil.javabaseproject.databinding.ActivityMainBinding;
+import com.mobile.tandil.javabaseproject.mvp.contract.ParkingContract;
+import com.mobile.tandil.javabaseproject.mvp.view.base.ActivityView;
+
```
Comment (SartiMau) [inline app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/ParkingView.java:7]: remove this extra space please, let only one

Comment (AndresConfortiGL) [reply app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/ParkingView.java:7]: done

**Review by SartiMau — APPROVED** [review]

**Review by DiegoDiazGlobant — APPROVED** [review]

**Pull request merged**